theme/themes/pxt/globals/site.variables

```diff
@@ -176,6 +176,7 @@
 @sidebarTutorialColor: @lightGrey;
 
 @homeScreenBackground: #ecf0f1;
+@cloudProjectCardBackground: #58AFBA;
```

Comment:
This css variable name is very specific. Might be better to make it categorical, like `@simulatorBackground` (or whatever makes sense).

Reply:
hmmm yeah I was thinking about that, but I was hesitant to give it a grand name when the color is just used once for now.